components/TimeBarChart.vue

```diff
@@ -121,14 +133,24 @@ const options: ThisTypedComponentOptionsWithRecordProps<
 > = {
   created() {
     this.canvas = process.browser
-    this.dataKind =
-      this.$route.query.embed && this.$route.query.dataKind === 'cumulative'
-        ? 'cumulative'
-        : 'transition'
+    // init dataKind
+    if (!this.$route.query.embed) {
+      this.dataKind = 'transition'
+    } else {
+      switch (this.$route.query.dataKind) {
+        case 'cumulative':
+        case 'transitionByDayOfTheWeek':
+          this.dataKind = this.$route.query.dataKind
+          break
+        default:
+          this.dataKind = 'transition'
+      }
+    }
```

Comment:
else節もあるので、次のようにしたほうがわかりやすいかと思います。

```suggestion
    if (this.$route.query.embed) {
      switch (this.$route.query.dataKind) {
        case 'cumulative':
        case 'transitionByDayOfTheWeek':
          this.dataKind = this.$route.query.dataKind
          break
        default:
          this.dataKind = 'transition'
      }
    } else {
      this.dataKind = 'transition'
    }
```

Reply:
ご確認ありがとうございます。適用いたします。    
（当初条件を連続させないように上記のように書きましたが、elseで結局条件連続していてご指摘の通り二重否定（!のelse）になっていることに気づいたため）